bin/i18n/sync-hourofcode-in

```diff
@@ -7,24 +7,36 @@ require 'fileutils'
 
 def pull_in_files_for_translation
   loc_dir = "i18n/locales/source/hourofcode"
+  Dir.mkdir(loc_dir) unless Dir.exist?(loc_dir)
   orig_file = "pegasus/sites.v3/hourofcode.com/i18n/en.yml"
   FileUtils.cp(orig_file, loc_dir)
 
-  orig_dir = Dir["pegasus/sites.v3/hourofcode.com/public/*.md"]
+  orig_dir = Dir["pegasus/sites.v3/hourofcode.com/public/*.{md,md.partial}"]
   orig_dir.each do |file|
     loc_dir = "i18n/locales/source/hourofcode"
+    if File.extname(file) == '.partial'
```

Comment:
Won't the `extname` be `.md.partial`?

Reply:
It might be cleaner to perform this rename entirely in the "up" step, rather than here

Reply:
When I was experimenting with it it only took the last extension.

I'm not sure what we'd gain from this being in the up step? That step is pretty well contained to just the crowdin call.

Reply:
Oh, what I mean is that we can do this rename purely with the crowdin config script changes to coerce `*.md.partial` files to be `%file_name%.md`

Reply:
Looks like there is a `dest` setting but it says that setting only works for single file name. The `type` parameter still forces a line ending that somehow suggest markdown.

https://support.crowdin.com/configuration-file/#uploading-files-to-specified-path-with-specified-type

Reply:
Oooh, I was definitely misunderstanding how the `translation` property works.

However, I also didn't know about the `type` property. Can we just use that to convince Crowdin to treat the `.md.partial` files as markdown?

Reply:
When I tried that I got errors 😞 
![Screenshot 2019-08-22 at 1 26 39 PM](https://user-images.githubusercontent.com/46464143/63547217-85526a00-c4e0-11e9-83f9-e3e43eff70b9.png)

Reply:
One has to wonder what the point of being able to specify filetype is, then. :upside_down_face: 

In that case, it seems like we _do_ need to do the `.md.partial` -> `.md` conversion in the "in" step, which means we don't need the entry in the `.yml` config file for dealing with `.partial`s.

Reply:
oh, which I now see you've already done!

Reply:
I was _so_ excited when I saw the `type` property just to be quite disappointed.